### PR TITLE
feat: add taglist limiter

### PR DESCRIPTION
**Pull request by nickfranciosi**
adds wrapper around tagList to limit number of visible tags

### CODE REVIEW

File: src/components/tagList/index.jsx [hunk 1]
```diff
@@ -24,12 +24,13 @@ const TagList = ({ children, rows, style }) => (
...
-        style: styles.tag,
-      })
-    ))}
+    <Style
+      scopeSelector=".TagList"
+      rules={{
+        ".Tag": styles.tag,
+      }}
+    />
+    {children}
```
Comment (thomasthesecond) [inline src/components/tagList/index.jsx]: What is the reason for this change?

File: stories/index.jsx [hunk 2]
```diff
@@ -2140,8 +2141,24 @@ storiesOf("Tag list", module)
...
+        <Tag href="#">World</Tag>,
+        <Tag href="#">Asia & the Pacific</Tag>,
+        <Tag href="#">Europe</Tag>,
+        <Tag href="#">Middle East & Africa</Tag>,
+        <Tag href="#">Editor’s pick</Tag>,
+        <Tag href="#">Brains</Tag>,
+        <Tag href="#">Mustard</Tag>,
+        <Tag href="#">Noodles</Tag>,
+        <Tag href="#">Ed’s pick</Tag>,
+      ]}
```
Comment (thomasthesecond) [inline stories/index.jsx]: Can these `Tag`s be passed as children instead?

File: src/components/tagListLimiter/index.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React, { PropTypes } from "react";
+import TagList from "../tagList";
+import Tag from "../tag";
+
+class TagListLimiter extends React.Component {
```
Comment (thomasthesecond) [inline src/components/tagListLimiter/index.jsx]: What do you think about extending `TagList` to accommodate this behavior? What if we just passed in the limit prop to `TagList` and added state? And if you don't pass a limit it just shows them all.

**Comment by nickfranciosi** [comment]
@thomasthesecond I moved this logic into the TagList. Thanks!
